### ISSUE TITLE
Semantic release 

### ISSUE THREAD

**Issue by estahn** [goreleaser/goreleaser#910]
**Is your feature request related to a problem? Please describe.**

TL;DR: Either use semantic-release generated GitHub Release Output or provide a changelog generator based on semantic-release ([Example of generated output](https://github.com/semantic-release/semantic-release/releases/tag/v16.0.0-beta.1%40beta)). 

I'm using the current workflow to trigger goreleaser ([CircleCI](https://github.com/hipages/php-fpm_exporter/blob/master/.circleci/config.yml)):

1. Merge to master --> Tests --> [Semantic Release](https://github.com/semantic-release/semantic-release)
2. Tag created by Sematic Release --> goreleaser

This has the advantage that I can use semantic-release and goreleaser together. Hence don't need to think what part of the version needs bumping.

The problem with this is that semantic-release creates a subjectively nice looking changelog from the "meta-data" in the git commit messages but goreleaser is overriding this in the second step. Imo goreleaser will need to attach information such as docker image version, so this is required. 

**Describe the solution you'd like**

- Copy the formatting functionality from semantic-release
- to be discussed...

**Comment by estahn** [comment]
@caarlos0 Will this keep the information about `Docker images`? Also, will I need to generate the title of the changelog?

**Comment by caarlos0** [comment]
> @caarlos0 Will this keep the information about `Docker images`? Also, will I need to generate the title of the changelog?

if I recall correctly, it keeps the docker info, yes... if by the title of the changelog you mean `## Changelog`, yes, you'll need to add it on your file...

**Comment by estahn** [comment]
@caarlos0 Great, thanks for your help! 🌞
